src/events/callback/handler.h

```diff
@@ -85,8 +85,10 @@ namespace Bald {
             }
 
             if(!handled) {
-                CORE_LOG_WARN("[Handler] Could not create Handler object because maximum number of ID's was reached");
+                CORE_LOG_WARN("[Handler] Could not create Handler object because maximum number of ID's was reached!");
```

Comment:
We either put full stops/commas/exclamation marks or we don't. From what I can see, until now we have not been using any punctuation after sentences.

Reply:
But you are free to pick one convention, but please check whether it is followed everywhere.